tests/shared/src/test/scala/spire/math/IntervalGeometricPartialOrderSuite.scala

```diff
@@ -22,8 +23,9 @@ class IntervalGeometricPartialOrderSuite extends munit.FunSuite {
   test("empty cannot be compared to [2, 3]") { assert(open(2, 2).partialCompare(closed(2, 3)).isNaN) }
   test("[2, 3] cannot be compared to empty") { assert(closed(2, 3).partialCompare(open(2, 2)).isNaN) }
   test("Minimal and maximal elements of {[1], [2, 3], [2, 4]}") {
-    val intervals = Seq(point(1), closed(2, 3), closed(2, 4))
-    assertEquals(intervals.pmin.toSet, Set(point(1)))
-    assertEquals(intervals.pmax.toSet, Set(closed(2, 3), closed(2, 4)))
+    val intervals = Seq[Interval[Int]](point(1), closed(2, 3), closed(2, 4))
+    // Scala 3 doesn't pick seqops automatically
+    assertEquals(seqOps(intervals).pmin.toSet, Set(point(1)))
+    assertEquals(seqOps(intervals).pmax.toSet, Set(closed(2, 3), closed(2, 4)))
```

Comment:
Ah, answers my question above. Can we fix this?

Reply:
I tried to fix it but couldn't find a reason. maybe the imports need to be more specific

Reply:
Huh. Ok that's fine, we can't fix everything.